chapters/operatorsandexpressions.tex

```diff
@@ -1415,6 +1415,20 @@ \section{Variability of Expressions}\label{variability-of-expressions}
 \end{lstlisting}
 \end{example}
 
+\subsection{Function Variability}\label{function-variability}
+
+The variability of function calls needs to consider both the variability of arguments directly given in the function and the variability of the used default arguments, if any.
+This is especially a concern for functions given as a short class, see \cref{inheritance-of-functions}.
+This has additional implications for redeclarations, see \cref{function-compatibility}.
+The purity of the function, see \cref{pure-modelica-functions}, does not influence the variability of the function call.
+
+\begin{nonnormative}
+The reason the variability ignores if functions is declared as \lstinline!impure! is that even in this case variability does not depend on the function.
+Consider a function reading an external file and returning some value from that file.
+Different uses can have the file updated before the simulation (as a parameter-expression), or during the simulation (as a discrete-time expression).
+Thus it depends on the use case and the specific file, not the function itself, and it would even be possible to update the file in continuous time (as part of an algorithm) and still use the same function.
+\end{nonnormative}
+
```

Comment:
Yes, I think rephrasing just the first sentence on line 1426 could be enough.

It would be nice to avoid the formulation "modify variability":
> The restrictions for calling functions declared as impure serve a similar purpose as the variability restrictions, see 12.3, and thus it is not necessary consider purity in the definition of variability.

Reply:
Looks good now.